event_registration_cancel_reason/views/event_registration_view.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+    <record model="ir.ui.view" id="view_event_registration_form">
+        <field name="name">Event registration form (with cancel reason)</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.view_event_registration_form"/>
+        <field name="arch" type="xml">
+            <field name="date_closed" position="after">
+                <field name="cancel_reason_id"/>
```

Comment:
Yes, but maybe you can make a PR over my branch to complete comments as I'm now doing another things.